src/components/FileIcon.tsx

```diff
@@ -290,29 +297,28 @@ function fileExtension(filename: ?string) {
   };
 }
 
-/**
- * @example ./examples/FileIcon.md
- */
```

Comment:
do we not need this comment block and md reference anymore? tbh not sure if we're using this to build docs examples

Reply:
Ah definitely, will get rid of this, thanks